packages/design-system-react/package.json

```diff
@@ -47,6 +47,7 @@
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
   },
   "dependencies": {
+    "@radix-ui/react-slot": "^1.1.0",
```

Comment:
Utilizes radix slot which we are planning on using throughout the design system react component library as outlined in the POC findings